puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/containers.groovy

```diff
@@ -0,0 +1,26 @@
+import groovy.json.JsonOutput
+
+def createTimestamp() {
+    (new Date()).format("yyyyMMdd-HHmm", TimeZone.getTimeZone('UTC'))
+}
+
+def triggerGithubBuilder(project_name, version, version_build) {
+    def inputs = JsonOutput.toJson([
+        ref: 'master',
+        inputs: [
+            project_name: project_name,
+            version: version,
+            push: 'true',
+            registry: 'xyz',
+            version_build: version_build
+        ]
+    ])
+
+    def command_parts = [
+        "curl -X POST -H 'Accept: application/vnd.github.v3+json' -u $GITHUB_USER:$GITHUB_TOKEN",
```

Comment:
Careful here, it may get logged. There's a secrets module as well to store these things.

Reply:
I know, it uses secrets and they were not leaked when I tested: 

![secrets](https://user-images.githubusercontent.com/2664090/98132764-12eb6100-1ebd-11eb-97de-6ee757c99384.png)